src/Functions/visitParamHas.cpp

```diff
@@ -21,28 +21,42 @@ using FunctionSimpleJSONHas = FunctionsStringSearch<ExtractParamImpl<NameSimpleJ
 
 REGISTER_FUNCTION(VisitParamHas)
 {
-    factory.registerFunction<FunctionSimpleJSONHas>(FunctionDocumentation{
-        .description = "Checks whether there is a field named field_name.  The result is UInt8.",
-        .syntax = "simpleJSONHas(json, field_name)",
-        .arguments
-        = {{"json", "The JSON in which the field is searched for.", {"String"}},
-           {"field_name", "The name of the field to search for.", {"const String"}}},
-        .returned_value = {"It returns 1 if the field exists, 0 otherwise."},
-        .examples
-        = {{.name = "simple",
-            .query = R"(CREATE TABLE jsons
+    FunctionDocumentation::Description description = R"(
```

Comment:
The [docs](https://clickhouse.com/docs/sql-reference/functions/json-functions#simplejson-visitparam-functions) make this comment about _multiple_ simpleJSON\* functions:

```ClickHouse has special functions for working with simplified JSON. All these JSON functions are based on strong assumptions about what the JSON can be. They try to do as little as possible to get the job done as quickly as possible.

The following assumptions are made:
[...]
```

I feel the comment is important. Will it survive the transition to generated docs?